src/api/PDFPage.ts

```diff
@@ -566,6 +570,63 @@ export default class PDFPage {
     this.node.wrapContentStreams(startRef, endRef);
   }
 
+  /**
+   * Scale the content of a page. This is useful after resizing an exisiting page.
+   * This scales only the content not the annots. See also: [[scaleAnnots]]
+   * ```js
+   * // bisect the size of the page
+   * p.setSize(p.getWidth() / 2, p.getHeight() / 2);
+   *
+   * // scale the content of the page down by 50% in x and y
+   * page.scaleContent(0.5, 0.5);
+   * ```
+   * @param x The factor by wich the x-axis for the content should be scaled (e.g. 0.5 is 50%)
+   * @param y The factor by wich the y-axis for the content should be scaled (e.g. 0.5 is 50%)
+   */
+  scaleContent(x: number, y: number): void {
```

Comment:
It would probably make sense to also call `setSize` within the high-level `scale` method. That way it becomes trivial for users to resize a page along with all of its contents.